### PR TITLE
New Pie Charts

### PR DESCRIPTION
**Pull request by Rachel-ais**
The only other thing to look into is adding a custom legend but the rest is done for now

### CODE REVIEW

File: projects/developer/src/app/common/services/app-config.service.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class AppConfigService {
             .then((data: any) => {
                 // loop over all keys in the appconfig.json file
                 Object.keys(data).forEach(key => {
+                    console.log('service', key, data[key]);
```
Comment (mheppner) [inline projects/developer/src/app/common/services/app-config.service.ts]: Can you remove pls

File: projects/developer/src/app/common/services/app-config.service.ts [hunk 2]
```diff
@@ -31,6 +32,7 @@ export class AppConfigService {
                     // set the camelcased version of the key in environments
                     environment[_.camelCase(key)] = value;
                 });
+                console.log('e', environment);
```
Comment (mheppner) [inline projects/developer/src/app/common/services/app-config.service.ts]: remove pls

File: projects/developer/src/app/data/ingest/component.ts [hunk 3]
```diff
@@ -3,6 +3,9 @@ import { Router, ActivatedRoute, Params } from '@angular/router';
 import { LazyLoadEvent, SelectItem } from 'primeng/primeng';
 import { MessageService } from 'primeng/components/common/messageservice';
 import { BreakpointObserver, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';
```
Comment (mheppner) [inline projects/developer/src/app/data/ingest/component.ts]: all of the changes to this file shouldn't be in this PR, I think you may have incorrectly merged from upstream

File: projects/developer/src/app/processing/batches/batches-component.ts [hunk 4]
```diff
@@ -3,6 +3,9 @@ import { Router, ActivatedRoute, Params } from '@angular/router';
 import { LazyLoadEvent, SelectItem } from 'primeng/primeng';
 import { MessageService } from 'primeng/components/common/messageservice';
 import { BreakpointObserver, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';
```
Comment (mheppner) [inline projects/developer/src/app/processing/batches/batches-component.ts]: same here, all the changes in this file shouldn't be here

File: projects/developer/src/app/processing/jobs/component.ts [hunk 5]
```diff
@@ -1,9 +1,12 @@
-import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnDestroy, OnInit, AfterViewInit, Output } from '@angular/core';
```
Comment (mheppner) [inline projects/developer/src/app/processing/jobs/component.ts]: check this file too

File: projects/developer/src/app/processing/recipes/component.ts [hunk 6]
```diff
@@ -3,6 +3,8 @@ import { Router, ActivatedRoute, Params } from '@angular/router';
 import { LazyLoadEvent, SelectItem } from 'primeng/primeng';
 import { MessageService } from 'primeng/components/common/messageservice';
 import { BreakpointObserver, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';
```
Comment (mheppner) [inline projects/developer/src/app/processing/recipes/component.ts]: also don't commit anything in this file

File: projects/developer/src/app/system/scans/component.ts [hunk 7]
```diff
@@ -3,6 +3,9 @@ import { Router, ActivatedRoute, Params } from '@angular/router';
 import { LazyLoadEvent } from 'primeng/primeng';
 import { MessageService } from 'primeng/components/common/messageservice';
 import { BreakpointObserver, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';
```
Comment (mheppner) [inline projects/developer/src/app/system/scans/component.ts]: same for this file

**Comment by Rachel-ais** [comment]
@cshamis  

We tried to add the labels within the chart but could not find a way for them to be readable any easier than the icons. 
Screen Shot 2020-02-04 at 6 56 54 AM

Even if we pulled the labels off the graph like discussed in the meeting we would have to shrink the pie chart so significantly to get them to fit. 

Screen Shot 2020-02-04 at 7 00 57 AM

**Comment by cshamis** [comment]
Ok, This is never going to work.  Cardinality to high for job types.  try mapping outer ring to spectrum of colors for PRIORITY of all jobs currently in queue.

Each distinct priority would get a different color on the Color ramp.  

If we can’t show the jobs themselves (which in a way is redundant I suppose since we have the job “blocks” list below...  then maybe we can show a breakdown of NEW information like priority which is also useful to know at a glance.

(And for love of all that is UI Holy, please kill that serif font!)

And thanks for being a sport and going back and forth with me.  —-You get sprint MVP again.

**Pull request merged**